mllib/src/test/java/org/apache/spark/ml/param/JavaTestParams.java

```diff
@@ -72,15 +72,31 @@ public JavaTestParams setMyStringParam(String value) {
     set(myStringParam_, value); return this;
   }
 
+  private DoubleArrayParam myDoubleArrayParam_;
+  public DoubleArrayParam myDoubleArrayParam() { return myDoubleArrayParam_; }
+
+  public double[] getMyDoubleArrayParam() { return getOrDefault(myDoubleArrayParam_); }
+
+  public JavaTestParams setMyDoubleArrayParam(double[] value) {
+    set(myDoubleArrayParam_, value); return this;
+  }
+
   private void init() {
-    myIntParam_ = new IntParam(this, "myIntParam", "this is an int param", ParamValidators.gt(0));
+    myIntParam_ = new IntParam(this, "myIntParam", "this is an int param",
+      ParamValidators.gt(0));
     myDoubleParam_ = new DoubleParam(this, "myDoubleParam", "this is a double param",
       ParamValidators.inRange(0.0, 1.0));
     List<String> validStrings = Lists.newArrayList("a", "b");
     myStringParam_ = new Param<String>(this, "myStringParam", "this is a string param",
       ParamValidators.inArray(validStrings));
+    myDoubleArrayParam_ =
+      new DoubleArrayParam(this, "myDoubleArrayParam", "this is a double param");
+
     setDefault(myIntParam_, 1);
+    setDefault(myIntParam_.w(1));
```

Comment:
minor: We should test `myIntParam()` instead of `myIntParam_`. The latter is private.